### PR TITLE
Feature/meru mc v2

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Added a Meru controller module that supports RADIUS disconnect

# Impacts
Nothing, just adds a module

# Code / PR Dependencies
None

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
(REQUIRED, but may be optional...)
## New Features
* Added a new Meru controller module that supports RADIUS RFC3576 (RADIUS Disconnect)

### CODE REVIEW

**Comment by julsemaan** [comment]
If you have a better name than MC_v2 that would also be good

File: lib/pf/Switch/Meru/MC_v2.def [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Tunnel-Type = 13
+Tunnel-Private-Group-ID = $vlan
+EOT
+acceptRole=<<EOT
+Filter-ID = $role
+EOT
+disconnect=<<EOT
+Calling-Station-Id = $mac
+EOT
+
```
Comment (jrouzierinverse) [inline lib/pf/Switch/Meru/MC_v2.def:16]: Add the following comment, please.
Then I will commit.
```
# Do not edit.
# Any changes will be lost on upgrade.	
```

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]

**Comment by jrouzierinverse** [comment]
A few suggestions for the name.
[MC Hammer](https://youtu.be/otCpCn0l4Wo)
MC Escher
MC Squared 
MC Zam
MC Jules

**Comment by extrafu** [comment]
MC Forti

**Comment by julsemaan** [comment]
I decided to go with the best of all suggestions: Forti_MC_Hammer

**Comment by extrafu** [comment]
We also forgot MC Circulaire ... he has some pretty epic songs that well qualify this module.

File: lib/pf/Switch/Meru/MC_v2.def (hunk 1 above)
Comment (julsemaan) [reply lib/pf/Switch/Meru/MC_v2.def:16]: Done

**Pull request merged**